euth/actions/signals.py

```diff
@@ -1,18 +1,75 @@
+from django.apps import apps
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
 from euth.actions import emails
+from euth.modules.models import Item
+from euth.projects.models import Project
 
 from . import verbs
 from .models import Action
 
 
+def notify_creator(instance):
+    if hasattr(instance.target, 'creator'):
+        creator = instance.target.creator
+        if creator.get_notifications and not creator == instance.actor:
+            emails.notify_users_on_create_action(instance, [creator.email])
+
+
+def notify_moderators(instance):
+    if str(instance.target_content_type) == 'project':
+        recipients = instance.target.moderators.all().values_list(
+            'email', flat=True)
+        emails.notify_users_on_create_action(instance, recipients)
+
+
+def add_action(sender, instance, created, **kwargs):
+    content_type = ContentType.objects.get_for_model(instance)
+    verb = verbs.CREATE if created else verbs.UPDATE
+    actor = instance.creator
+
+    action = Action(
+        actor=actor,
+        verb=verb,
+        action_object_content_type=content_type,
+        action_object_object_id=instance.pk,
+    )
+
+    if str(content_type) == 'Comment':
```

Comment:
Comments do also have a project property. So they probably don't need any special treatment. when getting the project:

```
if hasattr(instance, 'project'):
    project = instance.project
else:
   project = None
```

Should be sufficient for all Actionables